pkg/controller/iotproject/tenant_cleanup.go

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2019, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package iotproject
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/enmasseproject/enmasse/pkg/util/altclient"
+
+	"github.com/google/uuid"
+
+	"github.com/enmasseproject/enmasse/pkg/util/finalizer"
+	"github.com/enmasseproject/enmasse/pkg/util/install"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+
+	iotv1alpha1 "github.com/enmasseproject/enmasse/pkg/apis/iot/v1alpha1"
+)
+
+var (
+	tenantCleanupNamespace uuid.UUID = uuid.MustParse("20c0db1c-ffc8-11e9-800c-c85b762e5a2c")
```

Comment:
Any particular reason this is not a random value?

Reply:
It actually is … however it must be stable. The idea is that different names, with different namespaces result in different hashes. so NS1:"foo" gives a different named UUID than NS2:"foo".

Reply:
It actually is … however it must be stable. The idea is that different names, with different namespaces result in different hashes. so NS1:"foo" gives a different named UUID than NS2:"foo".

Reply:
Sounds good. Was just wondering about why that specific value.